### ISSUE TITLE
init command throws invalid GitHub url error

### ISSUE THREAD

**Issue by ge4human** [haydenbleasel/next-forge#189]
Hey! congrats on the launch. I've been experiencing this error since yesterday when trying to initialize the project.

<img width="682" alt="Screenshot 2024-11-06 at 2 16 46 AM" src="https://github.com/user-attachments/assets/26f2e75e-c8dc-444d-b25c-4efa22e1c52f">

**Comment by haydenbleasel** [comment]
I haven't seen this one before! Can you run the setup command directly and see what happens?

```
pnpm create next-app@latest prx --example https://github.com/haydenbleasel/next-forge
```

**Comment by ge4human** [comment]
@haydenbleasel still the same issue

```
westdabestdb@Gorkems-Mac-Studio Desktop % pnpm create next-app@latest prx --example https://github.com/haydenbleasel/next-forge
Found invalid GitHub URL: "https://github.com/haydenbleasel/next-forge". Please fix the URL and try again.
```

**Comment by haydenbleasel** [comment]
Okay, thanks for checking. That's very unfortunate, it seems your terminal can't access GitHub for some reason. Never seen this one before. But, you can circumvent this issue by just cloning the repo from GitHub, then running `scripts/setup.sh`.

**Comment by MinhOmega** [comment]
I have the same problem here:
![CleanShot 2024-11-13 at 09 56 50](https://github.com/user-attachments/assets/e86cbb42-47e3-4173-9417-d6cca24b8e78)
When cloning this repo and running `scripts/setup.sh` it still not working for me.
```
./scripts/setup.sh
Setting up next-forge...
Installing dependencies...
Scope: 2 of 21 workspace projects
Lockfile is up to date, resolution step is skipped
Packages: +314
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++

   ╭──────────────────────────────────────────────────────────────────╮
   │                                                                  │
   │                Update available! 9.11.0 → 9.12.3.                │
   │   Changelog: https://github.com/pnpm/pnpm/releases/tag/v9.12.3   │
   │                Run "pnpm add -g pnpm" to update.                 │
   │                                                                  │
   │         Follow @pnpmjs for updates: https://x.com/pnpmjs         │
   │                                                                  │
   ╰──────────────────────────────────────────────────────────────────╯

Progress: resolved 314, reused 314, downloaded 0, added 314, done

devDependencies:
+ @auto-it/all-contributors 11.3.0
+ @auto-it/first-time-contributor 11.3.0
+ @biomejs/biome 1.9.4
+ @repo/typescript-config 0.0.0 <- packages/typescript-config
+ turbo 2.2.3
+ typescript 5.6.3
+ ultracite 4.1.6

Done in 3.3s
Copying .env.example files to .env.local...
Deleting internal content...
rm: .github/FUNDING.md: No such file or directory
Setting up Prisma...

> next-forge@2.13.7 build /Users/minhvo/Desktop/Web/next-forge
> turbo build "--filter" "@repo/database"

turbo 2.2.3

• Packages in scope: @repo/database
• Running build in 1 packages
• Remote caching disabled
@repo/database:build: cache miss, executing de045ea0d8c1aa8c
@repo/database:build:
@repo/database:build:
@repo/database:build: > @repo/database@0.0.0 build /Users/minhvo/Desktop/Web/next-forge/packages/database
@repo/database:build: > prisma generate --no-hints
@repo/database:build:
@repo/database:build: sh: prisma: command not found
@repo/database:build:  ELIFECYCLE  Command failed.
@repo/database:build:  WARN   Local package.json exists, but node_modules missing, did you mean to install?
@repo/database:build: ERROR: command finished with error: command (/Users/minhvo/Desktop/Web/next-forge/packages/database) /Users/minhvo/.nvm/versions/node/v20.16.0/bin/pnpm run build exited (1)
@repo/database#build: command (/Users/minhvo/Desktop/Web/next-forge/packages/database) /Users/minhvo/.nvm/versions/node/v20.16.0/bin/pnpm run build exited (1)

 Tasks:    0 successful, 1 total
Cached:    0 cached, 1 total
  Time:    762ms
Failed:    @repo/database#build

 ERROR  run failed: command  exited (1)
 ELIFECYCLE  Command failed with exit code 1.
Setup complete! Deleting scripts folder...
```
Please take a look, here is my info:
- MacBook Pro M1 pro - MacOS 15.1
- NodeJS 20.16
- Pnpm 9.11.0
- npm 10.8.1
